src/tools/pkcs15-tool.c

```diff
@@ -1645,6 +1658,21 @@ static int list_apps(FILE *fout)
 	return 0;
 }
 
+
```

Comment:
clang-format suggested the following change:


```
-
```

<!-- identifier: clang-format -->
<!-- File: src/tools/pkcs15-tool.c -->